src/api/app/models/package_issue.rb

```diff
@@ -2,6 +2,11 @@ class PackageIssue < ApplicationRecord
   belongs_to :package
   belongs_to :issue
 
+  scope :of_open_issues, -> (owner_id) { joins(:issue).where(issues: { state: 'OPEN', owner_id: owner_id})}
```

Comment:
I would prefer ```open_issues_of_owner``` or ```open_issues_of_owner_id``` given it expects the owner id.

No strong opinion regarding chained or not chained:)